### DISCUSSION TITLE
Need help in picking hardware

### DISCUSSION THREAD

**Discussion by Goobertown43** [LostRuins/koboldcpp#1035] [Q&A]
Hello friends! I worked a little and saved some cash, and nvidia 4060 is on the discount near me, i can afford it but firstly i need help, i need a pc to put it in

I seriously thought about dell t7810 since its cpu's support avx2, has a shit load of ram and can fit multiple gpus if i cared to

If not dell t7810 then could anyone advise me for a workstation i could get second hand that would make more sense please?

@LostRuins pwease

**Comment by Goobertown43** [comment]
in case its useful, where i live i could also technically get dell t7810 but its more expensive :<

**Reply by Goobertown43** [reply]
misspelled, t5820

**Comment by LostRuins** [comment]
Any reasonably modern intel cpu will work great. The GPU is more important.

**Reply by Goobertown43** [reply]
so you think the dell t7820 would be a good idea?

**Reply by Goobertown43** [reply]
sorry i have autism so you gotta be a straight forward to me

**Reply by Goobertown43** [reply]
@LostRuins btw, how great of a bottleneck would be using gen 3 pcie? the dell i wanna buy has gen 3 not 4

**Comment by Goobertown43** [comment]
@LostRuins big question, i am kinda stupid so sorry if it doesnt make sense, but, https://ark.intel.com/content/www/us/en/ark/products/120476/intel-xeon-gold-6138-processor-27-5m-cache-2-00-ghz.html on there (the cpu i wanna buy) it says that it supports avx512, does it mean that it supports avx2 too? or do those two are not necesarly going in pairs?

**Reply by LostRuins** [reply]
AVX512 is newer but different from AVX2, but while that CPU you linked does indeed have AVX2, it seem way overkill. 

I don't know if the dell t7820 would be a good idea. It would certainly work, but there might be better options depending on your budget.

In general almost any CPU made within the last 5 years would be perfectly fine for AVX2.

**Reply by Goobertown43** [reply]
it seems like an overkill but there is a good reason for it, i found ir cheap, where i live there is quite a rich second hand market, so said cpu and dell t5820 is going together cheaper than my bike

**Reply by Goobertown43** [reply]
and @LostRuins if there are better options, could you please tell me about a potentially better workstation? a legit dell t5820 is the best cost to power ratio hence why i thought about it, but if you know of any better workstation to hunt, or a motherboard that i could then fully diy a pc from then please tell me

**Reply by Goobertown43** [reply]
best cost to power ratio from the one's i found at least

**Reply by Goobertown43** [reply]
and does the koboldcpp can use AVX512? you say it is newer but different, could you tell me a little what does it mean?

**Reply by LostRuins** [reply]
AVX512 is not supported in the precompiled binaries.

**Reply by Goobertown43** [reply]
but if i were to compile myself then it could be used?

**Reply by LostRuins** [reply]
Possibly, it has not been tested though. It would likely not provide any speedup as I doubt there are many ops using this instruction

**Reply by Goobertown43** [reply]
hmmm, ye its fine

but it is good to know said pc would work, it is an overkill but it would be useful as i dont buy it for just kobold, it will be used for the type of work i do, but it is nice to know kobold will work
